wdl/imp_sub.wdl

```diff
@@ -36,6 +38,7 @@ task plink_to_vcf {
     File joint_qc_exclude_variants
     File batch_qc_exclude_variants
     File exclude_samples
+    File force_impute_variants
```

Comment:
consider adding as optional variant and in command check if defined and touch or copy a file appropriately so all downstream commands will have empty file to work with in case not defined